samples/ipc/ipc_service/sample.yaml

```
@@ -1,52 +1,120 @@
+      - remote_CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=1
     extra_args:
-      OVERLAY_CONFIG=overlay-cpunet-sending.conf
-  sample.ipc.ipc_service.icmsg_default:
-    build_only: true
-    platform_allow: nrf5340dk_nrf5340_cpuapp
+      ipc_service_OVERLAY_CONFIG=rpmsg.conf
+      remote_OVERLAY_CONFIG=rpmsg.conf
+
+  sample.ipc.ipc_service.nrf5340dk_icmsg_default:
+    platform_allow:
+      - nrf5340dk_nrf5340_cpuapp
     integration_platforms:
       - nrf5340dk_nrf5340_cpuapp
-    tags: ipc
+    extra_configs:
+      - SB_CONFIG_REMOTE_BOARD='"nrf5340dk_nrf5340_cpunet"'
     extra_args:
-      CONF_FILE=prj_icmsg.conf
-  sample.ipc.ipc_service.icmsg_cpuapp_sending:
-    build_only: true
-    platform_allow: nrf5340dk_nrf5340_cpuapp
+      ipc_service_OVERLAY_CONFIG=icmsg.conf
+      ipc_service_DTC_OVERLAY_FILE=boards/nrf5340dk_nrf5340_cpuapp_icmsg.overlay
+      remote_DTC_OVERLAY_FILE=boards/nrf5340dk_nrf5340_cpunet_icmsg.overlay
+      remote_OVERLAY_CONFIG=icmsg.conf
+
+  sample.ipc.ipc_service.nrf5340dk_icmsg_cpuapp_sending:
+    platform_allow:
+      - nrf5340dk_nrf5340_cpuapp
     integration_platforms:
       - nrf5340dk_nrf5340_cpuapp
-    tags: ipc
+    extra_configs:
+      - SB_CONFIG_REMOTE_BOARD='"nrf5340dk_nrf5340_cpunet"'
+      - ipc_service_CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=1
+      - remote_CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=200000000
     extra_args:
-      CONF_FILE=prj_icmsg.conf
-      OVERLAY_CONFIG=overlay-cpuapp-sending.conf
-  sample.ipc.ipc_service.icmsg_cpunet_sending:
-    build_only: true
-    platform_allow: nrf5340dk_nrf5340_cpuapp
+      ipc_service_OVERLAY_CONFIG=icmsg.conf
+      ipc_service_DTC_OVERLAY_FILE=boards/nrf5340dk_nrf5340_cpuapp_icmsg.overlay
+      remote_DTC_OVERLAY_FILE=boards/nrf5340dk_nrf5340_cpunet_icmsg.overlay
+      remote_OVERLAY_CONFIG=icmsg.conf
+
+  sample.ipc.ipc_service.nrf5340dk_icmsg_cpunet_sending:
+    platform_allow:
+      - nrf5340dk_nrf5340_cpuapp
     integration_platforms:
       - nrf5340dk_nrf5340_cpuapp
-    tags: ipc
+    extra_configs:
+      - SB_CONFIG_REMOTE_BOARD='"nrf5340dk_nrf5340_cpunet"'
+      - ipc_service_CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=200000000
+      - remote_CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=1
+    extra_args:
+      ipc_service_OVERLAY_CONFIG=icmsg.conf
+      ipc_service_DTC_OVERLAY_FILE=boards/nrf5340dk_nrf5340_cpuapp_icmsg.overlay
+      remote_DTC_OVERLAY_FILE=boards/nrf5340dk_nrf5340_cpunet_icmsg.overlay
+      remote_OVERLAY_CONFIG=icmsg.conf
+
+  sample.ipc.ipc_service.nrf54h20pdk_cpuapp_cpuppr_icmsg:
+    platform_allow:
+      - nrf54h20pdk_nrf54h20_cpuapp
+    integration_platforms:
+      - nrf54h20pdk_nrf54h20_cpuapp
+    extra_configs:
+      - SB_CONFIG_REMOTE_BOARD='"nrf54h20pdk_nrf54h20_cpuppr"'
+    extra_args:
+      ipc_service_SNIPPET=nordic-ppr
+      ipc_service_OVERLAY_CONFIG=icmsg.conf
+      ipc_service_OVERLAY_CONFIG=boards/nrf54h20pdk_nrf54h20_cpuapp_cpuppr.conf
```

`icmsg.conf` is unused here (and in `remote`) but I suppose it's not needed.
Otherwise, this should've been:
```suggestion
      ipc_service_OVERLAY_CONFIG="icmsg.conf;boards/nrf54h20pdk_nrf54h20_cpuapp_cpuppr.conf"
```

---

done